packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -614,11 +614,16 @@ function onToggleNodesDisabled(ids: string[]) {
 	toggleNodesDisabled(ids);
 }
 
+function onClickNode() {
```

Comment:
Do you still need the separate `click:node` event now that you let `onSelectNode` always emit (also when there is no lastSelectedNode?